### PR TITLE
Renaming Interactive product assemblies...

### PR DESCRIPTION
**Pull request by KevinH-MS**
"Roslyn" -> "Microsoft" or "Microsoft.CodeAnalysis"

(I want to get this out of the way before we start "publishing" these in builds...)

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @tmat @ManishJayaswal @cston @amcasey @wschae

**Comment by cston** [comment]
LGTM

File: src/VisualStudio/VisualBasic/Repl/BasicVisualStudioRepl.vbproj [hunk 1]
```diff
@@ -22,6 +22,7 @@
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DocumentationFile>Microsoft.VisualStudio.VisualBasic.Repl.xml</DocumentationFile>
```
Comment (amcasey) [inline src/VisualStudio/VisualBasic/Repl/BasicVisualStudioRepl.vbproj]: Intentional?

**Comment by amcasey** [comment]
Am I missing the .sln changes?

**Comment by amcasey** [comment]
Oh, I see - the solution doesn't care about the name of the resulting DLL.

**Comment by amcasey** [comment]
:+1:

File: src/VisualStudio/VisualBasic/Repl/BasicVisualStudioRepl.vbproj (hunk 1 above)
Comment (KevinH-MS) [inline src/VisualStudio/VisualBasic/Repl/BasicVisualStudioRepl.vbproj]: Woops--that got auto-added.  Removed.

**Comment by ManishJayaswal** [comment]
:+1: 
This fixes #4195 . I will assign that to you Kevin to verify and close.

**Pull request merged**